Dockerfile.metadata.public

```diff
@@ -28,5 +28,10 @@ ENV SQLALCHEMY_DATABASE_URI sqlite:///sessions.db
 # FLASK_OIDC_CLIENT_ID - oidc client id
 # FLASK_OIDC_CLIENT_SECRET - oidc client secret
 
+FROM base as gremlin-release
+
+RUN  pip3 install -e .&& \
+     pip3 install -e .[gremlin]
```

Comment:
@callarelli93 So changes like this should probably not be submitted. We need to override those in our project in the same way we do for `Dockerfile.frontend.public`.